docs/manuals/userman/configuration/job-config.md

```diff
@@ -85,10 +85,17 @@ You can configure the *standalone* mode with the following config:
 To execute multiple `Job^`s simultaneously, you can set the *max_nb_of_workers* to an
 integer value greater than 1. That starts each `Job^` in a dedicated process with
 *max_nb_of_workers* as the limit of concurrent processes that can run simultaneously.
+The default value of *max_nb_of_workers* is 2.
 
-!!! note
+!!! warning
 
-    The default value of *max_nb_of_workers* is 2.
+    In *standalone* mode, when a subprocess is spawned, the code is re-executed except
```

Comment:
What about the code inside other pages?

Reply:
It will be rerun, but if the Core and GUI service are protected in the `if __name__ == "__main__"`, it won't matter since the subprocess will only execute the tasks